akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/StashingExample.java

```diff
@@ -0,0 +1,159 @@
+        this.taskId = taskId;
+        this.instruction = instruction;
+      }
+    }
+
+    public static final class EndTask implements Command {
+      public final String taskId;
+
+      public EndTask(String taskId) {
+        this.taskId = taskId;
+      }
+    }
+
+    public interface Event {
+    }
+
+    public static final class TaskStarted implements Event {
+      public final String taskId;
+
+      public TaskStarted(String taskId) {
+        this.taskId = taskId;
+      }
+    }
+
+    public static final class TaskStep implements Event {
+      public final String taskId;
+      public final String instruction;
+
+      public TaskStep(String taskId, String instruction) {
+        this.taskId = taskId;
+        this.instruction = instruction;
+      }
+    }
+
+    public static final class TaskCompleted implements Event {
+      public final String taskId;
+
+      public TaskCompleted(String taskId) {
+        this.taskId = taskId;
+      }
+    }
+
+    public static class State {
+      public final Optional<String> taskIdInProgress;
+
+      public State(Optional<String> taskIdInProgress) {
+        this.taskIdInProgress = taskIdInProgress;
+      }
+    }
+
+    public static Behavior<Command> createBehavior(PersistenceId persistenceId) {
+      return new TaskManager(persistenceId);
+    }
+
+    public TaskManager(PersistenceId persistenceId) {
+      super(persistenceId,
+        SupervisorStrategy.restartWithBackoff(Duration.ofSeconds(1), Duration.ofSeconds(30), 0.2));
+    }
+
+    @Override
+    public State emptyState() {
+      return new State(Optional.empty());
+    }
+
+    @Override
+    public CommandHandler<Command, Event, State> commandHandler() {
+      return
+        commandHandlerBuilder(State.class)
+        .matchCommand(StartTask.class, this::onStartTask)
+        .matchCommand(NextStep.class, this::onNextStep)
+        .matchCommand(EndTask.class, this::onEndTask)
+        .build();
+    }
+
+    private Effect<Event, State> onStartTask(State state, StartTask command) {
+      if (state.taskIdInProgress.isPresent()) {
+        if (state.taskIdInProgress.get().equals(command.taskId))
+          return Effect().none(); // duplicate, already in progress
+        else
+          return Effect().stash(); // other task in progress, wait with new task until later
+      } else {
+        return Effect().persist(new TaskStarted(command.taskId));
+      }
+    }
+
+    private Effect<Event, State> onNextStep(State state, NextStep command) {
+      if (state.taskIdInProgress.isPresent()) {
+        if (state.taskIdInProgress.get().equals(command.taskId))
+          return Effect().persist(new TaskStep(command.taskId, command.instruction));
+        else
+          return Effect().stash(); // other task in progress, wait with new task until later
+      } else {
+        return Effect().unhandled();
+      }
+    }
+
+    private Effect<Event, State> onEndTask(State state, EndTask command) {
```

Comment:
I think breaking up the docs to show this piece by piece to highlight the stashing would be more approachable. Not for this PR